### ISSUE TITLE
Optimize form of address in Certificate e-mail

### ISSUE THREAD

**Issue by hofmannc** [MPDL/KEEPER#401]
Testserver: qa keeper
Browser: firefox 74.0 (64-Bit) Chrome 80.0.3987.149
Version: Seafile 6.3.12
Actions: create new certificate -> receive info e-mail: "Dear None"

Expected result: Dear "username"

**Comment by vmakarenko** [comment]
reproduced

**Comment by vmakarenko** [comment]
fiixed

**Comment by hofmannc** [comment]
Testserver: qa keeper
Browser: firefox 74.0 (64-Bit) Chrome 80.0.3987.149
Version: Seafile 6.3.12

Observation: Form of adress now: "Dear hofmann@mpdl.mpg.de"

Question: all other emails start with "Guten Tag [Username] "

Expected result: Form of adress of all e-mails follw the same pattern

**Comment by vmakarenko** [comment]
fixed

**Comment by hofmannc** [comment]
Testserver: qa keeper
Browser: firefox 74.0 (64-Bit) Chrome 80.0.3987.149
Version: Seafile 6.3.12

Result: nok, both e-mails (Certificate and first archive e-mail) are still adressed with user's e-mail (after deleting cache)

**Comment by hofmannc** [comment]
Testserver: qa keeper
Browser: firefox 74.0 (64-Bit) Chrome 80.0.3987.149
Version: Seafile 6.3.12
Test time: 11:32

Result: nok, still adressed with user's email 
![grafik](https://user-images.githubusercontent.com/14921701/78011600-137ab280-7344-11ea-96b0-2bd5bb2d7bbb.png)

**Comment by vmakarenko** [comment]
Did you set your name in Settings->Name?

**Comment by hofmannc** [comment]
No, I actually did not set a name in "settings" -> retry and ok now.

--> But still - where does the user name in the other e-mails come from if no user name is given?

**Comment by MPDLRanger** [comment]
recheck, if still open, fix in next release
